frontend/.eslintignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-build
\ No newline at end of file
+build
+.eslintrc.js
```

Comment:
Getting this error, tried various workarounds but nothing worked
![Screenshot 2022-03-29 at 10 53 28](https://user-images.githubusercontent.com/32242596/160539203-e0d3d3ad-d308-4003-8b4d-02e1ef1f38a7.png)

Reply:
can you please ignore `eslintrc` and put it in `.eslintignore` _(with Axolo)_

Reply:
possibly this would work _(with Axolo)_

Reply:
Yes, already did. Added this comment to highlight why I did this :sweat_smile: _(with Axolo)_

Reply:
👍